test/e2e/autoscale_test.go

```diff
@@ -50,6 +50,8 @@ const (
 	// Concurrency must be high enough to avoid the problems with sampling
 	// but not high enough to generate scheduling problems.
 	containerConcurrency = 6
+	// Must match the const/config-map.
+	targetUtilization = 0.7
```

Comment:
Should we read this out of the configMap just like we do with `WaitForScaleToZero`?

Reply:
Done!